vue-app/src/views/ProjectList.vue

```diff
@@ -216,7 +218,14 @@ export default class ProjectList extends Vue {
 
   addMatchingFunds(): void {
     if (!this.$store.state.currentUser) {
-      return
+      return this.$modal.show(
```

Comment:
I think this entire method can be removed? I don't believe `addMatchingFunds` is used here.